LICENSE-binary

```diff
@@ -208,18 +208,18 @@ License Version 2.0:
 audience-annotations-0.5.0
 commons-cli-1.4
 commons-lang3-3.12.0
-jackson-annotations-2.13.3
-jackson-core-2.13.3
-jackson-databind-2.13.3
-jackson-dataformat-csv-2.13.3
-jackson-dataformat-yaml-2.13.3
-jackson-datatype-jdk8-2.13.3
-jackson-datatype-jsr310-2.13.3
-jackson-jaxrs-base-2.13.3
-jackson-jaxrs-json-provider-2.13.3
-jackson-module-jaxb-annotations-2.13.3
-jackson-module-scala_2.13-2.13.3
-jackson-module-scala_2.12-2.13.3
+jackson-annotations-2.13.4
+jackson-core-2.13.4
+jackson-databind-2.13.4
```

Comment:
2.13.4.2 ?

Reply:
It's such a shame, that I forgot to push the update. I have pushed it now.